modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java

```diff
@@ -0,0 +1,22 @@
+package net.legacyfabric.fabric.api.client.itemgroup;
+
+import net.legacyfabric.fabric.impl.item.group.ItemGroupExtensions;
+import net.minecraft.item.Item;
+import net.minecraft.item.itemgroup.ItemGroup;
+import net.minecraft.util.Identifier;
+
+public class FabricItemGroupBuilder {
+	private Identifier identifier;
```

Comment:
why are the fields here private?

Reply:
in the original fabric api it was like this =)

Reply:
you should look at this version of item group api I think.
https://github.com/FabricMC/fabric/tree/1.16/fabric-item-groups-v0